### PR TITLE
Vendor mimemagic to address the license change

### PR DESCRIPTION
**Pull request by mnyon-grandkru**
In your PR did you:

  - [X] Include a description of the changes? Vendored metamagic to deal with licensing issues
  - [X] Mention the issue the PR addresses? #4001

### CODE REVIEW

**Comment by rlgreen91** [comment]
@mnyon-grandkru merge conflicts

File: Gemfile [hunk 1]
```diff
@@ -119,7 +119,7 @@ gem 'social-share-button'
 gem 'select2-rails', '~> 4.0'
 
 # follower functionality
-gem 'acts_as_follower', github: 'tcocca/acts_as_follower'
+gem 'acts_as_follower'
```
Comment (rlgreen91) [inline Gemfile:122]: I believe the version listed on Rubygems has an issue with Rails 5+, which is why we switched to using the source here

**Review by rlgreen91 — COMMENTED** [review]
I really just have the one question about the change to the acts_as_follower gem

File: Gemfile (hunk 1 above)
Comment (mnyon-grandkru) [reply Gemfile]: Got it. Restoring github

**Review by rlgreen91 — APPROVED** [review]

**Pull request merged**